packages/@sanity/cli/src/actions/init-project/templates/shopify.ts

```diff
@@ -1,51 +1,69 @@
 import type {ProjectTemplate} from '../initProject'
 
-// @todo asset sources
-// @todo new document structure
-// @todo document actions
 const configTemplate = `
-import {defineConfig} from 'sanity'
+import {AssetSource, defineConfig, isDev} from 'sanity'
+
 import {deskTool} from 'sanity/desk'
-import {dashboard} from '@sanity/dashboard'
-import {media} from 'sanity-plugin-media'
-import {structure} from './deskStructure'
 import {schemaTypes} from './schemas'
+import {structure} from './desk'
+
+import {visionTool} from '@sanity/vision'
+import {colorInput} from '@sanity/color-input'
+import {imageHotspotArrayPlugin} from 'sanity-plugin-hotspot-array'
+import {media, mediaAssetSource} from 'sanity-plugin-media'
+import {customDocumentActions} from './plugins/customDocumentActions'
+
+const devOnlyPlugins = [visionTool()]
 
 export default defineConfig({
+  name: '%sourceName%',
+  title: '%projectName%',
+
+  projectId: '%projectId%',
+  dataset: '%dataset%',
+
   plugins: [
     deskTool({structure}),
+    colorInput(),
+    imageHotspotArrayPlugin(),
+    customDocumentActions(),
     media(),
-    dashboard({
-      widgets: [shopifyWidget()]
-    }),
+    ...(isDev ? devOnlyPlugins : []),
   ],
-  project: {
-    name: '%projectName%'
+
+  schema: {
+    types: schemaTypes,
   },
-  sources: [
-    {
-      name: '%sourceName%',
-      title: '%sourceTitle%',
-      projectId: '%projectId%',
-      dataset: '%dataset%',
-      schemaTypes
+
+  form: {
+    file: {
+      assetSources: (previousAssetSources: AssetSource[]) => {
+        return previousAssetSources.filter((assetSource) => assetSource !== mediaAssetSource)
+      },
     },
-  ],
-})
-`
+    image: {
+      assetSources: (previousAssetSources: AssetSource[]) => {
+        return previousAssetSources.filter((assetSource) => assetSource === mediaAssetSource)
+      },
+    },
+  },
+})`
 
 const shopifyTemplate: ProjectTemplate = {
   configTemplate,
-  importPrompt: 'Add a sampling of sci-fi movies to your dataset on the hosted backend?',
-  datasetUrl: 'https://public.sanity.io/moviesdb-2018-03-06.tar.gz',
   dependencies: {
-    '@sanity/dashboard': '^2.21.7',
+    '@sanity/asset-utils': '^1.3.0',
+    '@sanity/color-input': '^3.0.1',
     'lodash.get': '^4.4.2',
-    'pluralize-esm': '^9.0.2',
```

Comment:
Is there a reason we are not using `pluralize-esm` anymore?

Reply:
I think we should be using the`pluralize-esm` version here as it's the one we use elsewhere.